src/app/core/components/left-menu/left-menu.component.html

```diff
@@ -1,4 +1,4 @@
 <alg-left-header></alg-left-header>
```

Comment:
I have an issue with here:
1. As usual user, visit [this page](https://dev.algorea.org/branch/bugfix/left-menu-scroll-to-element/en/#/skills/by-id/1133373964112078654;path=3000,3002;attempId=0/details)
2. Under "skill for delete", I have the two children "Sub skill" and "sub skill 2" that are appended at the same level as their parent

![image](https://user-images.githubusercontent.com/16257159/183454031-63e0871b-d5f2-4ea9-97be-a0c0c0ae426f.png)

Maybe if you rebase your branch the problem will disappear, can you check?

Reply:
Yes, this problem has been fixed in master